ngmix/medsreaders.py

```diff
@@ -101,7 +101,7 @@ def get_mbobs(self, iobj, weight_type='weight'):
         return mbobs
 
 
-class MEDS(meds.MEDS):
+class MEDS(_MEDS):
```

Comment:
Shall we rename this class to something distinct from the original class name?  e.g. `NGmixMeds`?

Reply:
sure!